pkg/pushsync/pushsync.go

```diff
@@ -432,14 +401,27 @@ func (ps *PushSync) pushToClosest(ctx context.Context, ch swarm.Chunk, origin bo
 				}
 			}
 
-			ps.metrics.TotalSendAttempts.Inc()
+			// since we can reach into the neighborhood of the chunk
+			// act as the multiplexer and push the chunk in parallel to multiple peers
+			if ps.fullNode && swarm.Proximity(peer.Bytes(), ch.Address().Bytes()) >= ps.radiusChecker.StorageRadius() {
+				for ; parallelForwards > 0; parallelForwards-- {
+					retry()
+				}
+			}
 
+			action, err := ps.prepareCredit(ctx, peer, ch, origin)
```

Comment:
We should also think about the economics of this. Right now the multiplexer is bearing the cost of the parallel ops and he also doesnt store the chunk. Should he be getting reimbursed somehow? Maybe debiting the requestor peer for the multiplexing.

As he can reach the neighborhood, there is a chance that he will be reimbursed through retrievals in future. But it would be good to write all this down.

Reply:
naturally every peer has a likely chance of becoming a forwarder so it's balance out